tests/ui/rustc-print-info-issue-138612.rs

```diff
@@ -1,2 +0,0 @@
-//@ check-fail
-//@ compile-flags: /dev/null --print lints
```

Comment:
I was a bit confused by this test, it turns out we are checking for the `-Whelp` suggestion if the invalid print request `--print=lints` was specified.